ekm/storage.go

```diff
@@ -0,0 +1,31 @@
+package ekm
+
+import (
+	spectypes "github.com/bloxapp/ssv-spec/types"
+	"go.uber.org/zap"
+
+	"github.com/bloxapp/ssv/storage/basedb"
+)
+
+const (
+	prefix = "signer_data-"
+)
+
+// Storage represents the interface for ssv node storage
+type Storage interface {
+	SignerStorage
+	SpStorage
+}
+
+type storage struct {
+	SignerStorage
+	SpStorage
+}
```

Comment:
these are the same, why can't we just leave `storage`, we can change the name to `Storage`

Reply:
The `NewEKMStorage` function is designed to return interface, as it interacts with an external package, especially when setting the storage with `options.SetStorage(ekmStorage)`